base/style/breakpoints.less

```diff
@@ -8,6 +8,10 @@
 @width-xl: ~"(min-width: 1200px)";
 @xl-max: ~"(max-width: 1200px)";
 
+// Based on inspector "Laptop with MDPI screen" resolution
+@mdpi-min: ~"(min-width: 1280px)";
+@mdpi-max: ~"(max-width: 1280px)";
+
```

Comment:
Hm... most of the web runs on these:
https://getbootstrap.com/docs/5.0/layout/breakpoints/